### PR TITLE
[1LP][RFR] Adding ONE_PER_TYPE in ssui/services tests

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================

{{pytest: --long-running  cfme/tests/cloud_infra_common/test_cockpit_server.py cfme/tests/services/test_add_remove_vm_to_service.py cfme/tests/services/test_cloud_service_catalogs.py cfme/tests/services/test_different_dialogs_in_catalogs.py cfme/tests/services/test_myservice.py cfme/tests/services/test_provision_stack.py cfme/tests/services/test_request.py cfme/tests/services/test_service_catalogs.py cfme/tests/services/test_service_manual_approval.py cfme/tests/services/test_service_rbac.py cfme/tests/ssui/test_ssui_dashboard.py cfme/tests/ssui/test_ssui_myservice.py cfme/tests/ssui/test_ssui_service_catalogs.py }}

### CODE REVIEW

File: cfme/tests/ssui/test_ssui_service_catalogs.py [hunk 1]
```diff
@@ -16,9 +16,13 @@
     test_requirements.ssui,
     pytest.mark.long_running,
     pytest.mark.ignore_stream("upstream"),
-    pytest.mark.provider(gen_func=providers,
```
Comment (Yadnyawalkya) [inline cfme/tests/ssui/test_ssui_service_catalogs.py]: :+1: :100:

File: cfme/tests/ssui/test_ssui_service_catalogs.py [hunk 2]
```diff
@@ -16,9 +16,13 @@
...
-                         filters=[ProviderFilter(classes=[InfraProvider, CloudProvider],
-                                                 required_fields=['provisioning'])])
+    pytest.mark.provider(
+        selector=ONE_PER_TYPE,
```
Comment (Yadnyawalkya) [inline cfme/tests/ssui/test_ssui_service_catalogs.py]: :+1: :100:

**Comment by niyazRedhat** [comment]
In test_orchestration_template.py we are not testing with any provider.It is just templates testing.

File: cfme/tests/services/test_cloud_service_catalogs.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 from cfme.cloud.provider import CloudProvider
 from cfme.cloud.provider.gce import GCEProvider
 from cfme.cloud.provider.azure import AzureProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.services.service_catalogs import ServiceCatalogs
 from cfme import test_requirements
 from cfme.utils.blockers import BZ
```
Comment (ganeshhubale) [inline cfme/tests/services/test_cloud_service_catalogs.py]: @niyazRedhat 
I think ```from cfme import test_requirements``` should be above line 8

**Pull request merged**